docs/intro/parameters_search.rst

```diff
@@ -0,0 +1,67 @@
+Parameters search
+=================
+
+You can run cross-validation in DeepPavlov to select best parameters of your model.
+For this purpose you have to run special command 'paramserach'. for example:
+
+.. code:: bash
+
+    python deeppavlov/paramserach.py path_to_json_config.json --folds 5
```

Comment:
should be rewritten as `python -m deeppavlov.paramsearch ...` to be ran after installing from pip